src/business/dal/api/api.connection.ts

```diff
@@ -0,0 +1,48 @@
+import axios, { AxiosInstance } from 'axios';
+
+export default class Api {
```

Comment:
### ❗ Issue: Rename this class to `ApiConnection`

The file name is `api.connection` and I like it. To be consisted with filename u should rename the class :)